src/plugins/blocksmith/components/google-map-block-modal/getGoogleMapCoordinates.js

```diff
@@ -0,0 +1,26 @@
+/* globals GOOGLE_MAPS_API_KEY */
+const getGoogleMapCoordinates = async (str) => {
```

Comment:
@bhsiao55 - how does `getCoordinatesFromAddress` sound for the file name? We don't have to know what service we are using by looking at the file name.

Reply:
file name changed